index.d.ts

```diff
@@ -84,10 +84,10 @@ export default class OptionsSync {
 	/**
 	 * Any defaults or saved options will be loaded into the `<form>` and any change will automatically be saved via `chrome.storage.sync`.
 	 *
-	 * @param {HTMLElementTagNameMap | HTMLFormElement} selector - The `<form>` that needs to be synchronized or a CSS selector (one element).
+	 * @param {string | HTMLElementTagNameMap | HTMLFormElement} selector - The `<form>` that needs to be synchronized or a CSS selector (one element).
 	 * The form fields' `name` attributes will have to match the option names.
 	 */
-	syncForm: (selector: HTMLElementTagNameMap | HTMLFormElement) => void;
+	syncForm: (selector: string | HTMLElementTagNameMap | HTMLFormElement) => void;
```

Comment:
Allowing `string` is required, otherwise we cannot use selectors aside from basic element selectors. This is one of the overloads for `document.querySelector`, https://github.com/Microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L10604

Reply:
Sorry, yeah, re-released. I was under the impression that `HTMLElementTagNameMap` would replace `string` entirely but that only supports single-word element selectors.